### ISSUE TITLE
hoverctl status should tell you if running webserver or proxy mode

### ISSUE THREAD

**Issue by sherl0cks** [SpectoLabs/hoverfly#673]
It's actually rather hard to discern this information during runtime. There aren't even messages in the bootlog.

**Comment by JohnFDavenport** [comment]
@sherl0cks Can you make sure that you have configured hoverctl to use the remote target: https://docs.hoverfly.io/en/latest/pages/tutorials/advanced/remotehoverfly/remotehoverfly.html

You should then be able to use hoverctl status

**Comment by sherl0cks** [comment]
@JohnFDavenport I have no issue with `hoverctl status` and remote targets. `hoverctl targets` works well, nice job on that. the issue is with the output of the command, which as of 0.14.2, does not tell me whether hoverfly is running as a webserver or a proxy. See below example:

```
$ hoverctl start
Hoverfly is now running

+------------+------+
| admin-port | 8888 |
| proxy-port | 8500 |
+------------+------+

$ hoverctl status

+------------+----------+
| Hoverfly   | running  |
| Admin port |     8888 |
| Proxy port |     8500 |
| Mode       | simulate |
| Middleware | disabled |
+------------+----------+

$ hoverctl start webserver
Hoverfly is now running as a webserver

+----------------+------+
| admin-port     | 8888 |
| webserver-port | 8500 |
+----------------+------+

$ hoverctl status

+------------+----------+
| Hoverfly   | running  |
| Admin port |     8888 |
| Proxy port |     8500 |
| Mode       | simulate |
| Middleware | disabled |
```

**Comment by tommysitu** [comment]
Quite agree with the suggestion, we can change `Proxy port` to `Webserver port` if hoverfly is running in webserver mode, or print a new entry shows `Proxy type : reverse (webserver)`

**Comment by sherl0cks** [comment]
I'm in favor of a new entry, I think it's more declarative.

**Comment by tommysitu** [comment]
@sherl0cks this is implemented in the latest v0.15.0 release.
